src/main/java/pageObjects/Licenses.java

```diff
@@ -0,0 +1,264 @@
+    private WebElement purchaseDate;
+    @FindBy(id = "license_expiration_date_picker")
+    private WebElement expirationDate;
+    @FindBy(id = "license_type-selectList")
+    private WebElement licenseTypeList;
+    @FindBy(id = "license_type-selectList-pack")
+    private WebElement selectPackList;
+    @FindBy(id = "license_type-selectList-individual")
+    private WebElement selectIndividualList;
+    @FindBy(id = "license-locations-selectList")
+    private WebElement licenseLocationList;
+    @FindBy(id= "license-locations-selectList-romania")
+    private WebElement RomaniaList;
+    @FindBy(id = "license_dropdown_11111")
+    private WebElement licenceDropdown;
+    @FindBy(id = "license_edit_11111")
+    private WebElement licenceEditDropdown;
+
+    public void checkIfUserCanAddLicense() {
+        signInAsUser();
+        licenseButton.click();
+    }
+    public void successfulCreateLicense() {
+        signInAsAdmin();
+        licenseButton.click();
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("add-license")));
+        addLicenseButton.click();
+        wait.until(ExpectedConditions.urlToBe(Constants.LICENSE_DETAILS_URL));
+        nameField.sendKeys(LICENSE_NAME_WS);
+        purchaseDate.sendKeys((PURCHASE_DATE_WS));
+        expirationDate.sendKeys((EXPIRATION_DATE_WS));
+        saveLicense.click();
+        wait.until(ExpectedConditions.urlToBe(Constants.LICENSES_URL));
+    }
+
+    public void checkIfNameFieldIsMandatory() {
+        signInAsAdmin();
+        licenseButton.click();
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("add-license")));
+        addLicenseButton.click();
+        wait.until(ExpectedConditions.urlToBe(Constants.LICENSE_DETAILS_URL));
+        saveLicense.click();
+    }
+
+    public void createLicenseWithWhiteSpacesNameField() {
+        signInAsAdmin();
+        licenseButton.click();
+        wait.until(ExpectedConditions.urlToBe(Constants.LICENSES_URL));
+        addLicenseButton.click();
+        nameField.sendKeys(WHITE_SPACE_NAME_WS);
+        purchaseDate.sendKeys(PURCHASE_DATE_WS);
+        expirationDate.sendKeys(EXPIRATION_DATE_WS);
+        saveLicense.click();
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("error_toaster")));
+    }
+
+    public void addLicenseTooLongName() {
+        signInAsAdmin();
+        licenseButton.click();
+        addLicenseButton.click();
+        nameField.sendKeys(LICENSE_NAME_IS_TOO_LONG);
+        saveLicense.click();
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("error_toaster")));
+    }
+
+    public void createLicenseWithPackType() {
+        signInAsAdmin();
+        licenseButton.click();
+        wait.until(ExpectedConditions.visibilityOfElementLocated(By.id("add-license")));
+        addLicenseButton.click();
+        wait.until(ExpectedConditions.urlToBe(Constants.LICENSE_DETAILS_URL));
+        nameField.click();
```

Comment:
do we need to click on this field? becouse in others tests you did not click on it